eng/Versions.props

```diff
@@ -1,9 +1,9 @@
 <Project>
   <PropertyGroup>
     <!-- The .NET product branding version -->
-    <ProductVersion>9.0.0</ProductVersion>
+    <ProductVersion>10.0.0</ProductVersion>
```

Comment:
I can pull this branding update out into a separate PR if desired

Reply:
It's ok to include it in this PR.